portfolio/src/main/webapp/index.html

```diff
@@ -29,13 +29,21 @@ <h1>About Me</h1>
             of User Interfaces and Web Development. </p>
         <p> In my free time, you can either find me playing an instrument, reading, watching movies, swimming, 
             listening to music, taking pictures, or playing sudoku. I have a wide range of interests :) </p>
-        <p>Click <a href="/hello">here</a> to see content created by server code.</p>
+        
     </div>
 
     <div id="fact-gen">
         <h2>Click here to get a random fact about me</h2>
       <button id="button" onclick="addRandomGreeting()">Here</button>
       <div id="greeting-container"></div>
     </div>
+    <div id="fact-gen">
+        <h4> Click <a href="/hello">here</a> for a motivational message created using server response.</h4>
+        <h4>Click here to get a message from me directly on the page</h4>
+        <button id="button" onclick="addMotivationalMessage();">
```

Comment:
Maybe make the button id name more specific, like get-message-button. When you add more buttons, a generic name like button could be confusing

Reply:
Fixed